pkg/cmd/pipelinerun/describe.go

```diff
@@ -150,3 +160,31 @@ func hasFailed(pr *v1alpha1.PipelineRun) string {
 	}
 	return ""
 }
+
+type taskrunList []struct {
+	TaskrunName string
+	*v1alpha1.PipelineRunTaskRunStatus
+}
+
+func newTaskrunListFromMap(statusMap map[string]*v1alpha1.PipelineRunTaskRunStatus) taskrunList {
+
+	var trl taskrunList
+
+	for taskrunName, taskrunStatus := range statusMap {
+		trl = append(trl, struct {
```

Comment:
nit: consider making a `type` for the `struct{...}` instead of repeating it.

Reply:
that was intentional because it will not be used anywhere else and hence felt to having an anonymous struct would be better

Reply:
Agree with @sthaha, it would be nice to have it?

Reply:
done @hrishin @sthaha

Reply:
@girishramnani thanks for changing it. For the record, I had marked it as a `nit` meaning you don't really have to change it unless you feel convinced and won't stop the pr from merging.